test/functional/wallet_simulaterawtx.py

```diff
@@ -46,41 +46,87 @@ def run_test(self):
 
         # Add address1 as watch-only to w2
         w2.importpubkey(pubkey=w1.getaddressinfo(address1)["pubkey"])
-
         tx1 = node.createrawtransaction([], [{address1: 5.0}])
         tx2 = node.createrawtransaction([], [{address2: 10.0}])
 
         # w0 should be unaffected, w2 should see +5 for tx1
         assert_equal(w0.simulaterawtransaction([tx1])["balance_change"], 0.0)
         assert_equal(w2.simulaterawtransaction([tx1])["balance_change"], 5.0)
-
         # w1 should see +5 balance for tx1
-        assert_equal(w1.simulaterawtransaction([tx1])["balance_change"], 5.0)
+
+        tx1hex = node.decoderawtransaction(tx1)
+
+        details = [
+            {
+                "txid" : tx1hex["txid"],
+                "wallet_conflicts" : [],
+                "simulated_tx_conflicts" : []
+            },
+        ]
+        rpc_output = {
+            "balance_change" : 5.0,
+            "details" : details,
+            "duplicates" : []
+        }
+        assert_equal(w1.simulaterawtransaction([tx1]), rpc_output)
+
 
         # w0 should be unaffected, w2 should see +5 for both transactions
         assert_equal(w0.simulaterawtransaction([tx1, tx2])["balance_change"], 0.0)
         assert_equal(w2.simulaterawtransaction([tx1, tx2])["balance_change"], 5.0)
-
+        rpc_output = {
+            "balance_change" : 5.0,
+            "details" : details,
+            "duplicates" : []
+        }
         # w1 should see +15 balance for both transactions
-        assert_equal(w1.simulaterawtransaction([tx1, tx2])["balance_change"], 15.0)
+        details = [
+            {
+                "txid" : tx1hex["txid"],
+                "wallet_conflicts" : [],
+                "simulated_tx_conflicts" : []
+            },
+            {
+                "txid" : node.decoderawtransaction(tx2)["txid"],
+                "wallet_conflicts" : [],
+                "simulated_tx_conflicts" : []
+            },
+        ]
+
+        rpc_output = {
+            "balance_change" : 15.0,
+            "details" : details,
+            "duplicates" : []
+        }
+        assert_equal(w1.simulaterawtransaction([tx1, tx2]), rpc_output)
 
         # w0 funds transaction; it should now see a decrease in (tx fee and payment), and w1 should see the same as above
         funding = w0.fundrawtransaction(tx1)
         tx1 = funding["hex"]
         tx1changepos = funding["changepos"]
         bitcoin_fee = Decimal(funding["fee"])
-
-        # w0 sees fee + 5 btc decrease, w2 sees + 5 btc
+        # w0 sees (fee + 5) btc decrease, w2 sees + 5 btc
```

Comment:
In bf9deb407a8d73ac3b645b34c9bd6d28729a809c "rpc/wallet: Add details and duplicate section for simulaterawtransaction"

Unrelated change